command/ibft/quorum/ibft_quorum.go

```diff
@@ -0,0 +1,69 @@
+package quorum
+
+import (
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/spf13/cobra"
+)
+
+func GetCommand() *cobra.Command {
+	ibftQuorumCmd := &cobra.Command{
+		Use:     "quorum",
+		Short:   "Specify the block number after which quorum optimal will be used for reaching consensus",
+		PreRunE: runPreRun,
+		Run:     runCommand,
+	}
+
+	setFlags(ibftQuorumCmd)
+	setRequiredFlags(ibftQuorumCmd)
+
+	return ibftQuorumCmd
+}
+
+func setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&params.genesisPath,
+		chainFlag,
+		fmt.Sprintf("./%s", command.DefaultGenesisFileName),
+		fmt.Sprintf(
+			"the genesis file to update. Default: ./%s",
```

Comment:
No need to put the `. Default:` section here, Cobra will auto-generate it for you  👍

Reply:
[Removed](https://github.com/0xPolygon/polygon-edge/pull/549/commits/f6534cdc3fbdfcc85db3a0fc2de7a26605df6c4c)

Reply:
You forgot the `./%s`. Cobra will fill it like `flag description (default: ...)`

Reply:
Got it. 👍 

[Removed](https://github.com/0xPolygon/polygon-edge/pull/549/commits/da6b48688e0dc6bd07bc6a4b9a583e841046ed2f)